composer.scripts.json

```diff
@@ -0,0 +1,33 @@
+{
+  "scripts": {
+    "grumphp": [
+      "Composer\\Config::disableProcessTimeout",
```

Comment:
Is this really needed?

Reply:
I think yes, because this script can take a time and just fall by timeout